.github/workflows/build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       runs-on: ${{ matrix.os }}
       strategy:
         matrix:
-          os: [ubuntu-22.04, ubuntu-24.04]
+          os: [ubuntu-22.04, ubuntu-24.04, macos-13, macos-latest] # macos-13 is the last Intel mac version
```

Comment:
I'd be fine with just macos-latest ... but i suppose doesnt hurt to include intel chipset too.

Reply:
I think it make sense to use the Intel Mac for the people at SfG who have them.

As we're a public repo, it doesn't cost us any money to run this so there's no real harm in testing more broadly.

Reply:
Sure sure. I could check to see if anybody still is running the Intel chips -- I can't remember off the top of my head. Once we're sure we're all on ARM we could probably axe it and save a few cpu cycles...